docs/user_guides/debugging/backtrace.rst

```diff
@@ -0,0 +1,44 @@
+Backtrace
+==========
+
+``backtrace``
+-------------
+
+Let's consider the work of the command ``backtrace`` in the following example ``numba_dppy/examples/debug/simple_dppy_func.py``:
+
+.. literalinclude:: ../../../numba_dppy/examples/debug/simple_dppy_func.py
+    :lines: 15-
+    :linenos:
+
+Run debugger:
+
+.. code-block:: bash
+
+    export NUMBA_DPPY_DEBUGINFO=1
+    export NUMBA_OPT=1
```

Comment:
Why we use NUMBA_OPT=1 every time, but recommend users to debug on no optimization level? May be it better to use "0" level?